src/views/Billing/TabsView.vue

```diff
@@ -0,0 +1,96 @@
+<script setup>
+  import ContentBlock from '@/templates/content-block'
+  import PageHeadingBlock from '@/templates/page-heading-block'
+  import TabPanel from 'primevue/tabpanel'
+  import TabView from 'primevue/tabview'
+  import Tag from 'primevue/tag'
+  import ListView from './PaymentListView.vue'
+
+  import { ref } from 'vue'
+
+  import { useRoute, useRouter } from 'vue-router'
+
+  const route = useRoute()
+  const router = useRouter()
+  const activeTab = ref(0)
+
+  const props = defineProps({
+    paymentServices: { type: Object, required: true }
+  })
+
+  const mapTabs = ref({
+    bills: 0,
+    payment: 1
+  })
```

Comment:
```suggestion
  const TABS_MAP = {
    bills: 0,
    payment: 1
  }
```
não precisa de reatividade